Library/Homebrew/utils/curl.rb

```diff
@@ -127,6 +127,8 @@ def curl_check_http_content(url, user_agents: [:default], check_content: false,
     return "The URL #{url} is not reachable"
   end
 
+  return "#{url} permanently redirects to #{details[:final_301]}" if url != details[:final_301]
```

Comment:
Can this use `permanent_redirect` or something rather than assuming the reader will know what 301 means in this context (or even what a 301 HTTP status code means)? Thanks!

Reply:
I initially had something like `redirect_url` but didn't like it may be ambiguous between 301 permanent redirects and 302 found redirects. I've changed it to `permanent_redirect` now and pushed the changes.